src/Http/Controllers/ForgotPasswordController.php

```diff
@@ -18,6 +19,7 @@ class ForgotPasswordController extends Controller
     public function __construct()
     {
         $this->middleware(RedirectIfAuthenticated::class);
+        $this->middleware(Localize::class);
```

Comment:
This can be an array, like `$this->middleware([RedirectIfAuthenticated::class, Localize::class]);`

Reply:
Learning as I go. Thanks Erin. Fixed this.